android/Bottari/data/src/main/java/com/bottari/data/local/bottari/BottariDao.kt

```diff
@@ -0,0 +1,29 @@
+package com.bottari.data.local.bottari
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy.Companion.REPLACE
+import androidx.room.Query
+import com.bottari.data.model.local.bottari.BottariEntity
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface BottariDao {
+    @Query("SELECT * FROM Bottaries ORDER BY createdAt DESC")
+    fun fetchBottaries(): Flow<List<BottariEntity>>
+
+    @Query("SELECT * FROM Bottaries WHERE id = :id")
+    fun fetchBottari(id: Long): Flow<BottariEntity>
```

Comment:
코드 래빗이 알려준 것처럼 수정이 필요해 보이네요!


```suggestion
    fun fetchBottari(id: Long): Flow<BottariEntity?>
```